### ISSUE TITLE
Random Roll Simulator Button for Unique and Rare Templates

### ISSUE THREAD

**Issue by therber2** [PathOfBuildingCommunity/PathOfBuilding#3966]
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

Not directly, but yes.
The problem: sometimes it is tempting, when theory crafting a build you're thinking about in POB, to exaggerate the end result by creating unrealistic hypothetical gear..... similar to the idea of creating a build in pob with a hypothetical level 100 character for a league start character.

An example of a good solution you guys have already done within the software are the  lvl70 required Rare gear templates.
Those allow you to quickly create modest items that will more likely yield you a conservative / more realistic expectation of what you could expect from your build idea in game.

### Describe the solution you'd like

Just to extrapolate further on the already implemented....let's call them: "Realistic expectation templates" (for Rare gear).
What if under calcs tab we can have a drop down menu to select "Level of Expectation for Simulated Gear" > 'Optimal Rolls' , 'Semi Optimal' , 'Normal Odds' , 'Semi - Conservative ' , 'Conservative'  < where these threshholds would be referring simply to nothing other than fixed affix value rolls for Unique templates and Tier Values for Rare items you're either crafting or loading a saved Rare gear template. 

The UI would be adding a new button Under the "Items" tab over to the Right around the buttons: "Add to build" , "Edit" , "Cancel" when you have loaded any gear template (eg. Unique item, Any rarity craftable base, presaved Rare gear template)

> After double clicking either a Unique of a Rare item template from the searchable database, header titled: 'Import from:' 
> Add a button (again after opening the temple (eg. double clicked and opened temple for 'Crown of Eyes, Hubris Circlet' called 'Simulate' or 'Simulate RNG Drop' or 'Random roll item' 

>You would already have selected your level of RNG conservativism. "Normal" Being the default selection where "Normal" would refer, simply to the Median of values a mod can roll. One step up or down ("Semi-Optimistic or "Semi-Conservative") would be calculated as an RNG roll of the full range of values for the mod followed by adding or subtracting the value of the median value minus say 1 standard deviation.

So an example let's do for the hypothetical setting of "Semi-Conservative" in the calcs tab the RNG Simulation Confidence
For the Mod: '#% increased Energy Shield' for "New Item Hubris Circlet" .....

Given:
Example: RNG of the full range, randomly rolls:  88%
The Standard Deviation of the range of possible values (population) = 25.98 

Output:
Again we have 'Semi Conservative' selected so SUBTRACT 1 Std dev from the RNG roll
= 88 - 25.98
= 62% increased energy shield

I am just arbritrarily defining the break points for levels of RNG confidence.
Maybe steps by half of a standard deviation is better or just small steps and more incremental options under "Calc"

The Final Output upon clicking the proposed "Simulate" button would roll all mods together for one single output of course.

### Describe alternatives you've considered

Implementing 'Craft of Exile' api into the items tab?

### Additional context

TLDR: This will ultimately just be the same as Divine Orb'ing an item with POB with the option to slant the result towards lucky, or much more usefully slightly towards "Unlucky" to ultimate encourage realistic expectations in your general theory / build crafting!

**Comment by Wires77** [comment]
What you're looking for is outside the scope of PoB.  I would suggest just sticking with Craft of Exile for crafting simulation.  There is even an emulator now, I've found that lets you choose your mod tiers and roll divine orbs on it: https://www.craftofexile.com/?b=46&m=chaos&ob=both&v=d&a=e&l=a&lg=8&bp=y&as=0&bld={}&ggt=|&ccp={}

**Comment by therber2** [comment]
Ok. Thanks.
